### PR TITLE
DOCS-1065: Add Viam Triton server module information to modular resources examples

### PR DESCRIPTION
**Pull request by sguequierre**
* Add all information specified in GitHub, make configuration page

For @abe-winter or @acmorrow SME review-- some confusion on my end about built-in MLModel attributes for control card (`"model_path"`, `"label_path"`, and `"num_threads"`) vs attributes I made a table for like `model_name`-- `model_path` and `model_repository_path` are not the same thing behind-the-scenes right? If they are, please let me know what from my explanation is wrong and I can correct.

### CODE REVIEW

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:166]:
```suggestion
Now, use the `viam:mlmodelservice:triton` modular service to deploy MLModels to your robot.
```

**Review by abe-winter — COMMENTED** [review]
over my head! flagged one weird apt-get thing, otherwise will leave it to the actual expert

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:27]: That seems fine to me, done!

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [inline docs/extend/modular-resources/examples/triton.md:20]: The link here is to a specific version, but really it just needs to be Jetpack 5, so https://developer.nvidia.com/embedded/jetpack is probably better. Eventually we are going to require JetPack 6.

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [inline docs/extend/modular-resources/examples/triton.md:72]: There is no config builder for the triton module. The config on that card is specific to the tflite_cpu service. I do not know how that is supposed to be addressed: the space of modules is open so we can't know how to make a fancy config for each one.

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [inline docs/extend/modular-resources/examples/triton.md:74]: Like the above, this isn't actually true. The triton module has its own different attributes that must be configured.

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [inline docs/extend/modular-resources/examples/triton.md:165]: This sentence seems out of place.

**Review by acmorrow — CHANGES_REQUESTED** [review]
Looks pretty good. I think the only major issue is about the config card, since it doesn't actually work with the module.

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:20]:
```suggestion
1. [Jetpack 5](https://developer.nvidia.com/embedded/jetpack)
```

Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:20]: Sounds good to me, thanks! Changed

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:165]:
```suggestion
Now, use the `viam:mlmodelservice:triton` modular service to make use of MLModels on your robot.
```

Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:165]: Does `make use of` sound better? I guess deploy is misleading as it doesn't happen here. Can just remove sentence outright but want some kind of intro

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:72]: @acmorrow Ahhh gotcha. That could be confusing to users for now-- do you think lines 74-75 update is helpful?

Comment (acmorrow) [reply docs/extend/modular-resources/examples/triton.md:72]: Yeah, I think that's the right direction. I might make it even stronger and say "are not applicable for this module, and should be left blank".

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [inline docs/extend/modular-resources/examples/triton.md:72]: I think the `png` should be dropped.

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:52]:
```suggestion
  "version": "0.4.0"
```

Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:52]: Ok that makes sense, updating to `0.4.0`

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:74]:
```suggestion
Note that the parameters shown, `"model_path"`, `"label_path"`, and `"num_threads"` are not applicable for this module, and should be left blank.
```

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:72]: Hmmm- I see how that makes sense but then how will users know what card they should be ignoring? I think it might help for confusion to have the png and more explicitly explain

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:72]: @acmorrow Sounds good, done!

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:28]: gotcha, thank you!! For my information-- how do you identify which version was pulled, is there a command? Can add that in but I am not good with docker @acmorrow

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:107]:
```suggestion
The <file>config.pbext</file> file must exist, but at least for TensorFlow models it can be empty.
For other types of models, please consult the [Triton Server model repository documentation](https://docs.nvidia.com/deeplearning/triton-inference-server/user-guide/docs/user_guide/model_repository.html) for details on the correct contents of the <file>config.pbext</file> file.
```

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:107]: Sounds good, added

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:114]:
```suggestion
After creating your model repository, configure the attributes to deploy your model on your robot.
The following attributes are available for the MLModel service `viam:mlmodelservice:triton`:
```
@acmorrow re. call to action-- would this be correct? Apologies if confusion

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:172]:
```suggestion
Now, use the `viam:mlmodelservice:triton` modular service to perform inference with the machine learning models available in the Triton service's model repository on your robot.
```

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:172]: perfect, added!

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:114]: Added suggestion in comment @acmorrow LMK what you think, want to make sure I accurately understand

**Review by npentrel — APPROVED** [review]
This looks good to me as long as the SMEs approve!

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [reply docs/extend/modular-resources/examples/triton.md:28]: Sadly, there doesn't seem to be a good and simple way to do this. There are ways to do it but I'm worried the cure is worse, etc. I think I'd say something like "run `docker images ghcr.io/viamrobotics/viam-mlmodelservice-triton`" and examine the output to find the exact tag associated with `latest`".

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [inline docs/extend/modular-resources/examples/triton.md:58]: Let's add some text right below this that says "replace the value of the `version` field with the value you determined above with the `docker images` command".

File: docs/extend/modular-resources/examples/triton.md
Comment (acmorrow) [reply docs/extend/modular-resources/examples/triton.md:114]: That seems fine.

**Review by acmorrow — APPROVED** [review]
This LGTM once the suggested changes are applied. I left some notes on how to help the user identify `latest` and a suggested addition. But I don't think another round of review is required.

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:34]:
```suggestion

Examine the output to find the exact tag associated with latest.
Use this as `"version"` in [configuration](#configuration).
```

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:61]:
```suggestion

Replace the value of the version field with the value you determined above with the `docker pull` command.
```

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [reply docs/extend/modular-resources/examples/triton.md:114]: cool, added!

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:65]:
```suggestion

Add the following to your `"services"` array:
```

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:64]:
```suggestion
Replace the value of the `"version"` field with the value you determined above with the `docker pull` command.
```

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:37]:
```suggestion

Then, if you haven't done so already, create a new robot in [the Viam app](https://app.viam.com).
```

**Review by acmorrow — APPROVED** [review]
LGTM!

File: docs/extend/modular-resources/examples/triton.md
Comment (sguequierre) [inline docs/extend/modular-resources/examples/triton.md:192]:
```suggestion
You can now connect this vision service to a [transform camera](/components/camera/transform/), or get detections programmatically through one of Viam's [client SDKs](/program/).
```

**Pull request merged**
